### DISCUSSION TITLE
Screen brightness jumps when moving between apps

### DISCUSSION THREAD

**Discussion by menesescarlos** [MonitorControl/MonitorControl#1269] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

On my DELL screen, when switching between app screens, the screen brightness jumps high for a very short period of time (1 second); the screen brightness then returns to normal but it is annoying to have my monitor brightness jump every time I move between app screens.

### Steps to reproduce

1. Open Monitor control
2. Move between app screens (full screens)

### Expected behavior

Monitor brightness to not jump

### Anything else?

See video for issue: https://www.loom.com/share/479bcb9e496b4f0da320149cfed298e2

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 
- Mac model: Macbbok pro 14
- MonitorControl version:4.1.0
- Monitor(s):Dell
- Apple Silicon/M1 (yes or no): YES
```

**Comment by kynemre** [comment]
Yeap, I have a similar issue. When I switch tabs in Chrome, the brightness goes to max and doesn't back to normal.

Environment Info:
```
- macOS version: Ventura 13.1
- Mac model: Macbook Air 2020
- MonitorControl version: 4.1.0
- Monitor(s): Philips 272E2F
- Apple Silicon/M1 (yes or no): YES
```

**Comment by waydabber** [comment]
The brief change happens when you are using software dimming and disable gamma table manipulation. In this case MC uses a dark overlay to dim the screen, however this overlay is not present when there is a transition among desktops. This is normal behavior in this case.

**Comment by kynemre** [comment]
I have a Philips monitor (272E2FA/00). In my case, the screen brightness suddenly goes to the maximum and stays. I noticed this issue is not related to the application. It was fixed when I turned off SmartContrast in the monitor settings.
